jedi/api/__init__.py

```diff
@@ -708,7 +708,6 @@ class Interpreter(Script):
     :param namespaces: A list of namespace dictionaries such as the one
         returned by :func:`globals` and :func:`locals`.
     """
-    _allow_descriptor_getattr_default = True
```

Comment:
I thought default was True, so I didn't realize I had functionality changed the default somehow.

>The only alternative IMO would be to use a default like settings.interpreter_allow_descriptor_getattr

So have the Interpreter consume the new setting on creation *vs. right now it is a bit _global_ and changes any existing interpreter object) ?

That makes sense to me. I think that means as long as I make the jedi setting before starting ipython, I'd be ok.

Reply:
I'm not sure I understand your question. You might need to rephrase.